legend-engine-xt-relationalStore-grammar/src/main/java/org/finos/legend/engine/language/pure/grammar/to/HelperRelationalGrammarComposer.java

```diff
@@ -237,7 +237,13 @@ private static String renderElementWithJoins(ElementWithJoins elementWithJoins,
         StringBuilder builder = new StringBuilder();
         if (!elementWithJoins.joins.isEmpty())
         {
-            builder.append(LazyIterate.collect(elementWithJoins.joins, HelperRelationalGrammarComposer::renderJoinPointer).makeString(" > "));
+            String joins = LazyIterate.collect(elementWithJoins.joins.subList(0,1), HelperRelationalGrammarComposer::renderFirstJoinPointer).makeString(" > ");
```

Comment:
Could we make this more readable. Isn't this really just on join so the `makeString(" > ")` doesn't really come into effect?

Something like 
`
JoinPointer firstJoinPointer = ..
builder.append(renderFirstJoin(firstJoinPointer). 

... then continue if you if statement below. You dont need to concatenate. Leverage the string builder to append as needed.